### ISSUE TITLE
[Question] Compatibility with Anima NXT

### ISSUE THREAD

**Issue by daerSeebaer** [LamaDiLuce/polaris-opencore#83]
We use older and newer swords in our training. We recently update all the older swords with Anima EVO in them to OpenCore 3.1.0. However, the newer swords with the Anima NXT in it show an installed build number of "NXT_1.1.0", which indicates that they are not running OpenCore.

My questions resulting from this are:
- Is OpenCore compatible with Anima NXT?
- If not, where do I get the source code and/or firmware upgrades for them?

**Comment by f4iey** [comment]
No, the anima NXT runs on an STM32 microcontroller, which is a completely different hardware. The opencore project only supports compatible hardware for EVO (e.g Teensy 3.1/3.2).

And no, the anima NXT firmware is not yet available to the public and the source code will sadly be proprietary.

For now, you need to wait or contact the devs.

**Comment by daerSeebaer** [comment]
> No, the anima NXT runs on an STM32 microcontroller, which is a completely different hardware. The opencore project only supports compatible hardware for EVO (e.g Teensy 3.1/3.2).

Thanks for the explanation.

> And no, the anima NXT firmware is not yet available to the public and the source code will sadly be proprietary.

Is there a reason why the code will be proprietary? If the code for the Anima EVO is public, why not for the NXT?

> For now, you need to wait or contact the devs.

I was under the impression that the developers of Lama di Luce saber software can be reached exactly here. If that is not the case, where can I contact them?

**Comment by mrseeker** [comment]
Having the same question, since it seems that the sabers sold by LamaDiLuce are sold with version NXT_1.1.0

**Comment by f4iey** [comment]
> Having the same question, since it seems that the sabers sold by LamaDiLuce are sold with version NXT_1.1.0

Unfortunately, Lamadiluce has chosen to lock the codebase down for the NXT hardware. We cannot do anything about that. We will never have control on the firmware anymore

Again, this project (which is technically at the end of its life) *only applies to Teensy 3.x* hardware. And is NOT compatible with the anima NXT

`polaris-opencore` and `nxt` and completely different projects with different approaches.

**Comment by mrseeker** [comment]
Since I am a huge fan of the open-source and hacker community, are there any specs of the old hardware available? Like, if I was to be able to build a custom PCB with all components, what would be the dimensions that i would need to adhere to?
